Vagrantfile

```diff
@@ -0,0 +1,41 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
```

Comment:
It would be nice to add some brief documentation to your Readme to mention that this exists, what dependencies it has, and how it can be run